src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,31 @@
+.stars_section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  &_reviews {
+    display: flex;
+    align-items: flex-end;
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
If `start` is BEM block [checklist p8](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)